### PR TITLE
merging PR #103

### PR DESCRIPTION
**Pull request by urbagal**
PR request 103. Made small changes to run snpeff  in nextflow tower

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -60,8 +60,13 @@ params {
     save_reference             = true
     save_alignment             = true
     sample_ploidy              = 1
+<<<<<<< HEAD
```
Comment (hseabolt) [inline nextflow.config:63]: Check here (and also lines 63,66, and 69 for Git tags, e.g. <<<< HEAD, ====, and >>>> upstream/master.  These are added by Git during merging and we will want to remove them here.

File: modules/local/downsample_rate.nf [hunk 2]
```diff
@@ -8,9 +8,16 @@ process DOWNSAMPLE_RATE {
         'quay.io/biocontainers/python:3.8.3' }"
 	
     input:
+<<<<<<< HEAD
```
Comment (hseabolt) [inline modules/local/downsample_rate.nf:11]: There are a number of Git merging lines in this file that need to be removed.

File: subworkflows/local/bwa-pre-process.nf [hunk 3]
```diff
@@ -41,6 +41,10 @@ workflow BWA_PREPROCESS {
     
 
     SEQKIT_PAIR(reads)
+<<<<<<< HEAD
```
Comment (hseabolt) [inline subworkflows/local/bwa-pre-process.nf:44]: Similar to above, need to remove Git diff lines

File: subworkflows/local/snpeff.nf [hunk 4]
```diff
@@ -21,6 +21,7 @@ workflow SNPEFF {
 
     SNPEFF_ANN (
         vcf,
+        params.snpeffconfig,
```
Comment (hseabolt) [inline subworkflows/local/snpeff.nf:24]: `params.snpeffconfig` may need to be interpolated here with ${ }, but I'm not 100% sure.  If this has already been tested and confirmed working, then ignore this comment.

**Review by hseabolt — CHANGES_REQUESTED** [review]
Git merge elements like `<<<< HEAD`, `====`, and `>>>> upstream master` appear in several files here, we will need to get those removed prior to merging.  Other changes look fine to me w.r.t. documentation updates and minor tweaks to the code.  I am under the impression that these code changes have already been manually tested and are confirmed to be working as expected.

File: subworkflows/local/snpeff.nf (hunk 4 above)
Comment (urbagal) [reply subworkflows/local/snpeff.nf:24]: Yes, this has been tested and confirmed its working

**Review by hseabolt — APPROVED** [review]
Approving merge following removal of Git tags

**Pull request merged**